src/server/templates/target.js

```diff
@@ -0,0 +1,28 @@
+import favicon from './favicon';
+
+
+export default ({title, markup, page, props, script}) => `
+	  <!DOCTYPE html>
+	  <html>
+		<head>
+		  <title>${title ? `'${title} – BookBrainz'` :
+		     'BookBrainz – The Open Book Database'}</title>
+		  <link rel="stylesheet" href='/stylesheets/style.css' />
+		  <link rel="stylesheet" href='/stylesheets/react-virtualized.css' />
+		  <link rel="stylesheet"
+		    href='/stylesheets/react-virtualized-select.css' />
+		  <meta name='viewport' content='//react-virtualized.css' />
+		  ${favicon}
```

Comment:
`favicon` is imported as a function that returns a template string but it is never invoked. I recommend combining `favicon.js` into `target.js` and adding it as a constant for the time being.